css/my_portfolio.css

```diff
@@ -43,29 +43,156 @@ body {
   position: absolute;
 }
 
+.nav-link-border {
+  border-bottom: 1px solid #6f6c6b;
+}
+
 header {
   display: flex;
   justify-content: space-between;
-  position: relative;
   width: 100%;
   background: #3c3a39;
   height: 4em;
-  padding: 10%;
+  padding: 7%;
+  position: fixed;
+  z-index: 1;
 }
 
 .logo {
   color: #fff;
 }
 
 .logo-link {
-  color: white;
+  color: #fff;
+  text-decoration: none;
+  font-weight: bold;
+  opacity: 0.9;
+}
+
+.logo-link:hover {
+  color: #fff;
+  opacity: 1;
+  margin-top: -5px;
+  animation: logocolor 9s 1 linear;
+  font-size: 20px;
+}
+
+@keyframes logocolor {
+  100% {
+    filter: hue-rotate(360deg);
+    text-shadow: none;
+  }
+
+  0% {
+    filter: hue-rotate(0deg);
+    text-shadow: 0 0 10px #f55800, 0 0 1px #f55800, 0 0 10px #f55800, 0 0 20px #f55800, 0 0 30px #f55800, 0 0 40px #f55800;
+  }
+}
+
+.navbar {
+  width: 100%;
+  height: 100%;
+  background: #3c3a39;
+  position: fixed;
+  right: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 0;
+  transition: right cubic-bezier(1, 0, 0, 1);
+}
+
+.hamburger-menu {
+  width: 20px;
+  height: 16px;
+  background: #3c3a39;
+  position: fixed;
+  top: 25px;
+  right: 20px;
+  cursor: pointer;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.line {
+  width: 100%;
+  height: 3px;
+  border: 2px solid #acb7c3;
+  transition: all 0.8s;
+}
+
+.change {
+  right: 0;
+}
+
+.change .line-1 {
+  transform: rotateZ(-405deg) translate(-5px, 3px);
+}
+
+.change .line-2 {
+  opacity: 0;
+}
+
+.change .line-3 {
+  transform: rotateZ(405deg) translate(-5px, -4px);
+}
+
+.nav-list {
+  text-align: left;
+  position: absolute;
+  top: 10%;
+  left: 0%;
+}
+
+.nav-item {
+  list-style: none;
+  margin: 25px;
+  width: 130%;
+}
+
+.nav-link {
```

Comment:
- Kindly provide the requested design by the style guide on Figma
**_This is the requested design:_**


![Screen Shot 2021-12-13 at 16 41 25](https://user-images.githubusercontent.com/22258274/145900714-8aaf4675-68c8-4b2d-b038-49229f0bcc53.png)



**_Yours:_**
![Screen Shot 2021-12-13 at 16 40 34](https://user-images.githubusercontent.com/22258274/145900764-eaff344f-5809-4ded-a721-5bbd609782dd.png)

Reply:
@carlos-ssh, Thank you for the comment. Just to inform you nothing with the design. I am sure I met all the requirements. So this is not worth to be rejected.🤞